<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MoreSetting</class>
 <widget class="QWidget" name="MoreSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>489</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>When desktop is hidden</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="autoBox">
       <item>
        <property name="text">
         <string>continue playing</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>pause</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Video FPS</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fpsbox">
       <item>
        <property name="text">
         <string>default</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>30</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>60</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Decoder</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="hwdecBox">
       <item>
        <property name="text">
         <string>auto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gpu</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>vaapi</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>vdpau</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>none</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>other</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hwdecEdit">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_10">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>VO</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="voBox">
       <item>
        <property name="text">
         <string>libmpv</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>opengl</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>opengl-cb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>other</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="voEdit">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Show desktop icon</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="desktopShowCombox">
       <item>
        <property name="text">
         <string>true</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>false</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Place on top of the original desktop</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="topBox">
       <item>
        <property name="text">
         <string>true</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>false</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Desktop transparency </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_name">
       <property name="text">
        <string>xx</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="desktop_transparency">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Wallpaper transparency </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="back_transparency">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>99</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Event penetration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="eventBox">
       <item>
        <property name="text">
         <string>false</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>true</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Spark Desktop font color</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fontBox">
       <item>
        <property name="text">
         <string>default</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>white</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>black</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>blue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>yellow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>darkGray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>lightGray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>gray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>green</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>cyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>magenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>transparent</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>darkRed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>darkGreen</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>darkBlue</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>darkCyan</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>darkMagenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>darkYellow</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QPushButton" name="okBtn">
       <property name="text">
        <string>Confirm</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
